### PR TITLE
feat:  Add UnVerified role removal after verification

### PR DESCRIPTION
**Pull request by nihalxkumar**
- Add functionality to remove UnVerified role after successful verification
- Rename CodeModal to OTPModal
- Add section comments in OTPModal
- Fix one missed out change from last commit of renaming "Enter Code" to "Enter OTP"

### CODE REVIEW

**Review by sattwyk — CHANGES_REQUESTED** [review]
Please change the order of actions when handling roles. First, remove the "Unverified" role and then add the "Verified" role. This way, you avoid the situation where the "Verified" role is added but the "Unverified" role has not been removed. Additionally, include a check to see if the user has the "Unverified" role before making any changes.

File: bot/bot.py [hunk 1]
```diff
@@ -167,26 +173,38 @@ async def on_submit(self, interaction: discord.Interaction):
             if self.code.value != correct_code:
                 raise ValueError("Incorrect OTP")
 
+            # Add Verified Role
             verified_role = discord.utils.get(interaction.guild.roles, name="Verified")
```
Comment (sattwyk) [inline bot/bot.py:185]: here change the order

Comment (nihalxkumar) [reply bot/bot.py:185]: Fixed

Comment (sattwyk) [reply bot/bot.py:185]: thank u good ser

**Review by sattwyk — APPROVED** [review]
LGTM! - there might me some edge cases I gotta figure out but it's good for now

**Pull request merged**